### ISSUE TITLE
Recoverable Error from Translatable_Transformation::transformCheckboxField()

### ISSUE THREAD

**Issue by UndefinedOffset** [silverstripe/silverstripe-translatable#213]
It seems like #166 has come back in SilverStripe 3.2, using the latest latest from the 2.1 branch Translatable causes a crash when the form contains a checkbox field. The particular instance where I'm running into this issue specifically is on a UserForm page.

```
Argument 1 passed to Translatable_Transformation::transformCheckboxField() must be an instance of CheckboxField, instance of CheckboxField_Readonly given
```

**Comment by stealerr** [comment]
I am experiencing the same thing

**Comment by brasileric** [comment]
I experience the same problem.

**Comment by Sanderha** [comment]
Has this being fixed? I experience this issue aswell.

**Comment by tractorcow** [comment]
Ah yes, fixed with https://github.com/silverstripe/silverstripe-translatable/pull/215.

We'll fix this when we come to release 2.1.1 (or 2.2.0).
